Firestore/core/src/firebase/firestore/util/warnings.h

```diff
@@ -61,6 +61,14 @@
 
 #define SUPPRESS_END() _Pragma("GCC diagnostic pop")
 
+#elif defined(__GNUC__)
```

Comment:
If we're going to treat GCC completely separately, should the pragmas above refer to clang by name?

Reply:
I considered that, but decided to leave as is for now. Note that up on line 40, we still lump gcc/clang together, so we need to use GCC there, and if we're using GCC there, it makes sense to use it here too.

(The alternative is to also split at line 40; grouping clang/gcc there isn't buying us *too* much. Especially since this if block is the one more likely to grow.)